website/views.py

```diff
@@ -142,27 +141,8 @@ def index():
         pass
 
     user_id = get_current_user_id()
-    if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
-            )
-            .annotate(Count('nodes'))
-            .filter(nodes__count__gte=INSTITUTION_DISPLAY_NODE_THRESHOLD)
-            .order_by('name').only('_id', 'name', 'logo_name')
-        )
-        dashboard_institutions = [
-            {'id': inst._id, 'name': inst.name, 'logo_path': inst.logo_path_rounded_corners}
-            for inst in all_institutions
-        ]
-
-        return {
-            'home': True,
-            'dashboard_institutions': dashboard_institutions,
-        }
+    if user_id:
+        return send_from_directory(ember_osf_web_dir, 'index.html')
```

Comment:
Please leave all the legacy dashboard stuff in place and use waffle to choose between legacy dashboard and Ember dashboard.

Reply:
I believe I need a little more than this. As in, do you want a switch that is indefinitely set as either true or false, or a sample, or a flag?